deps/src/main/java/com/microsoft/azure/sdk/iot/deps/twin/TwinCollection.java

```diff
@@ -66,10 +66,12 @@
  * </pre>
  *
  * <p> This class exposes the Twin collection with or without metadata as a Map here
- *     user can gat both the value and the metadata. For instance, in the above TwinCollection,
+ *     user can get both the value and the metadata. For instance, in the above TwinCollection,
  *     {@link #get(Object)} for <b>Color</b> will return <b>White</b> and the {@link #getTwinMetadataFinal(String)}
  *     for <b>Color</b> will return the Object TwinMetadata that contain {@link TwinMetadata#getLastUpdated()}
- *     that will returns the {@code Date} <b>2017-09-21T02:07:44.238Z</b> and {@link TwinMetadata#getLastUpdatedVersion()}
+ *     that will returns the {@code Date} <b>2017-09-21T02:07:44.238Z</b>, {@link TwinMetadata#getLastUpdatedBy()}
```

Comment:
Why do we have a date example and word it like we'll return this exact date. Are we trying to say we'll return a date in this format?

Reply:
I read it as "return date and for example 2017-09-21(date format)". What do you think as a better way to describe it?

Reply:
I have added "for example" before this value, also added examples for the fields I added.